tests/mocks/rpc-handler.ts

```diff
@@ -12,6 +12,7 @@ export class RPCHandler implements HandlerInterface {
   private _networkName: string;
   private _env: string = "node";
 
+  private _rpcTimeout: number = 500; // ms
```

Comment:
For slow Internet 500 ms might not be enough. I don't think there should be any timeout. Just test all of them at the same time?

Reply:
I understand what you are saying here but for the UBQ usecase it should be.

- There are 8 RPCs for Gnosis, 5 at runtime on average
- There are ~28 for Mainnet, 8 at runtime on average

and that's with the 500ms timeout. When you start to play with not-so-popular networks with lesser quality RPCs then it becomes a problem as seen when using Filecoin for example.

I've seen reqs take longer than 3-4 seconds, and it just clogs the system up unnecessarily with RPCs that will in essence never be hit as they are the weakest of the bunch every time. Those 5 & 8 are always the same RPCs consistently

Reply:
I don't think we need to concern ourselves with memory overhead of open tcp connections until it becomes a performance issue.

Reply:
added a rough benchmark test and tested it on a local `pay.ubq` instance

- with the default there is a ~3s - 3.5s loading period till everything is populated
- with 500ms it's about ~2s - 2.3s